torch/_library/utils.py

```diff
@@ -215,6 +215,67 @@ def zip_schema(
     return
 
 
+def normalize_args_kwargs(
+    schema: _C.FunctionSchema, args: tuple[Any, ...], kwargs: dict[str, Any]
```

Comment:
You should annotate schema with ParamSpec so you can properly type the args and kwargs if you can. At the very least, the tuple here can be annotated with a TypeVarTuple to preserve the types within the tuple.